assignment4/assignment4.js

```diff
@@ -0,0 +1,36 @@
+var fs = require('fs');
+var data1, data2;
+fs.readFile('file1.txt', function (err, data) {
+  if (err) {
+      return console.error(err);
+  }
+  data1=data.toString();
+    fs.appendFile('file2.txt', data1 , function(error) {
+      if (error) {
+        console.log('Error:- ' + error);
+        throw error;
+      }
+      fs.readFile('file2.txt', function (err, data) {
+        if (err) {
+          return console.error(err);
```

Comment:
In the same situation above, where u are checking for error, `throw` statement is used and here `return` statement is used. Is there reason for this. Consistency is also important.